webpack.config.js

```diff
@@ -49,7 +51,8 @@ var config = [{
     './modules/help_editor/js/columnFormatter.js': './modules/help_editor/jsx/columnFormatter.js',
     './modules/brainbrowser/js/Brainbrowser.js': './modules/brainbrowser/jsx/Brainbrowser.js',
     './modules/data_integrity_flag/js/index.js': './modules/data_integrity_flag/jsx/index.js',
-    './modules/imaging_uploader/js/index.js': './modules/imaging_uploader/jsx/index.js'
```

Comment:
Why you removed this from here?
'./modules/imaging_uploader/js/index.js'

Reply:
it got moved to line 54 below